examples/miscellaneous/miscellaneous_ex12/miscellaneous_ex12.C

```diff
@@ -397,14 +415,14 @@ void assemble_shell (EquationSystems & es,
       // First compute element data at the nodes
       std::vector<Point> nodes;
       for (unsigned int i=0; i<elem->n_nodes(); ++i)
-        nodes.push_back(elem->node_ref(i));
+        nodes.push_back(elem->reference_elem()->node_ref(i));
```

Comment:
I realized that you have to pass points on the reference element to fe->reinit().
I'm a bit puzzled regarding why it was giving a correct solution before.